### ISSUE TITLE
Programs on modrinth

### ISSUE THREAD

**Issue by IZeeXeS** [modrinth/knossos#809]
### Describe the solution you'd like

Add a page for programs 

### Additional context

Minecraft related programs typically scattered around many sites, it would be useful for me and others to have a single place for it

**Comment by HyperSoop** [comment]
out of scope for modrinth sadly

**Comment by triphora** [comment]
Not really. This is the sort of thing Project Ogre is meant to address

**Comment by HyperSoop** [comment]
What is that?

**Comment by heipiao233** [comment]
Programs? Like NBTEditor?

**Comment by MMK21Hub** [comment]
Yeah, I assume that this refers to tools that operate outside of Minecraft but interact in Minecraft in some way. I'm not sure how they plan on presenting them in a coherent and organised manner, though.
